src/orbit-db-identity-provider.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+const IdentityProvider = require('./identity-provider')
+const Keystore = require('orbit-db-keystore')
+const keypath = './orbitdb/identity/keys'
+const type = 'orbitdb'
+
+class OrbitDBIdentityProvider extends IdentityProvider {
+  constructor (options = {}) {
+    super()
+    this._keystore = Keystore.create(options.keypath || keypath)
+  }
+
+  // Returns the type of the identity provider
+  static get type () { return type }
+
+  async getId (options = {}) {
+    const id = options.id
+    if (!id) {
+      throw new Error('id is required')
+    }
+
+    const keystore = this._keystore
+    const key = await keystore.getKey(id) || await keystore.createKey(id)
+    return key.getPublic('hex')
+  }
+
+  async signIdentity (pubKeyIdSig, options = {}) {
```

Comment:
One more small change request: rename the `pubKeyIdSig` to `data` as it is defined in the interface and in https://github.com/orbitdb/orbit-db-identity-provider/pull/13/files#diff-e17a429b402bcd7bc701a8c9caab648eR23. For sake of consistency across all files :)